### PR TITLE
Update architecture

### PR DESCRIPTION
**Pull request by onthebreeze**
Update architecture section with new 5-pillar overview and with supporting words for the product level and value chain level diagrams.

### CODE REVIEW

**Review by zachzeus — APPROVED** [review]
Looks good.

File: website/docs/specification/Architecture.md [hunk 1]
```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
...
+
+* The **Sustainability Vocabulary Catalog (SVC)** provides a framework to map sustainability knowledge across different standards, regulations and industry practices. It may not always answer the question but it provides a decentralised semantic governance model that allows mappings and corresponding value to grow over time and gaps to fixed as they are found. The SVC is a W3C DCAT conformant catalog of external sustainability standards and regulations. Mappings are defined using W3C SKOS and can be made either by UN working groups external standards **or** by external authorities to the UN catalog. This allows for a decentralised mapping effort that is far more scalable than depending on a small centralised team.
```
Comment (PatStLouis) [inline website/docs/specification/Architecture.md:64]: Great section!

File: website/docs/specification/Architecture.md [hunk 2]
```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
 
 ## Principles
 
+The architecture principles that guide the UNTP design are 
 
-## Each Actor
+|Name|Principle|Rationale|
+|--|--|--|
+|No dependency |UNTP should not require any collaboration or dependency between the issuer of DPPs and the consumer / verifier|Imposing such collaboration as a pre-requisite for action in a complex many-to-many ecosystem would essentially stall progress|
+|Unknown verifier |UNTP should not assume that that the consumer / verifier of UNTP data is known to the issuer, even when confidential data access is required|In a decentralised architecture with thousands of issuers, it would be impractical to register every authorised verifier with every issuer.|
```
Comment (PatStLouis) [inline website/docs/specification/Architecture.md:21]: > UNTP should not assume that that the consumer

UNTP should not assume that the consumer

File: website/docs/specification/Architecture.md [hunk 3]
```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
...
+|Any maturity|UNTP should not assume any technical maturity for verifiers|DPPs and other credentials must work equally for human and machine verifiers - otherwise an insurmountable complexity of knowing which customer has what capability would be required|
+|Legacy data carriers |UNTP should work with any carrier of a product identifier including iD barcodes, RFID, 2D codes, digital documents|1D barcodes and RFIDs are ubiquitous and will only be replaced slowly.  Uptake should not require manufacturers to re-instrument their production lines and printing processes|
```
Comment (PatStLouis) [inline website/docs/specification/Architecture.md:23]: This new work item proposition at the w3c caught my eyes and is directly related to this:
https://digitalbazaar.github.io/vc-barcodes/

File: website/docs/specification/Architecture.md [hunk 4]
```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
...
+* **The Digital Traceability Event (DTE)** provides a means to trace product batch data throughout the value chain. The DTE links input products (eg bales of cotton from the primary producer) to output products (eg woven cotton fabric). Therefore the DTEs provide a means to trace product provenance through manufacturing processes to discover an entire value chain. DTEs are only available when products are managed and traceable at batch level. DTEs provide links to reach deeper into the value chain which may contain commercially sensitive data and so may only be available to authorised roles. 
+
+All three UNTP data structures are designed to be extensible to meet the needs of specific industry sectors or jurisdictions. 
+
+### Finding the data
```
Comment (PatStLouis) [inline website/docs/specification/Architecture.md:46]: I like the term "discovering" but finding is also representative here.

File: website/docs/specification/Architecture.md [hunk 5]
```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
...
+* **Digital Identity Anchor (DIA)**. The issuers and subjects of Verifiable credentials are identified using W3C Decentralised Identifiers (DIDs) which provide a means to discover the cryptographic keys necessary to verify the credentials. However, DIDs are self-issued and do not ensure that the issuer is really who they say they are, only that the owner of the DID was certainly the issuer of the credential. The DIA is a verifiable credential issued by a trusted authority (eg a government agency) that links a DID to a known public identity such as VAT registration number. In this way, verifiers can be assured of the identity of issuers. The DIA also has a "scope" so that, for example a national accreditation authority can attest to the identity of a certifier but also specify the scope of the accreditation.
```
Comment (PatStLouis) [inline website/docs/specification/Architecture.md:57]: Great!

**Review by PatStLouis — APPROVED** [review]

**Pull request merged**